weldx/geometry.py

```diff
@@ -2534,6 +2534,8 @@ def _shape_triangles(
             Raster data of the shape
         offset :
             An offset that will be added to all indices.
+        closed_mesh :
+            If `True`, the side faces of the geometry will also be triangulated.
```

Comment:
From what you showed me before, do you think "front and back faces" is fitting here?

Reply:
It is probably more specific. I will adjust it and "smuggle it in" with another PR